integration_tests/__tests__/__snapshots__/failures.test.js.snap

```diff
@@ -150,14 +150,14 @@ exports[`works with node assert 1`] = `
     - Expected
     + Received
     
-      Object {
-        \\"a\\": Object {
-          \\"b\\": Object {
-    -       \\"c\\": 6,
-    +       \\"c\\": 5,
-          },
-        },
-      }
+     Object {
```

Comment:
It's suspicious that these are indented less by one space now. Why was this change made? It makes "Object" not line up with "Received" any longer.

Reply:
This space or no space confused me for a long while. We can go either way. Your choice!

Before this pull request the result changed according to CLI option:
* default `--no-expand` option uses hunk format from `diff` with no space between the first column marker and the actual line
* `--expand` option formatted the chunk lines from `diff` with a space there.

Reply:
I didn't realize that it was different based on cli options, it shouldn't be! I think two spaces (make it line up with "Received") makes more sense, and it would be good to be consistent.

Reply:
I like it better too. To make double sure I understand:
* The first column consists of -/+/space for removed/added/unchanged
* The second column consists of space to separate from line content and align with legend

Reply:
Yep! The first column may be empty space as well.